src/renderers/shared/fiber/isomorphic/ReactTypes.js

```diff
@@ -14,7 +14,9 @@
 
 import type { ReactCoroutine, ReactYield } from 'ReactCoroutine';
 
-export type ReactNode = ReactElement<any> | ReactCoroutine | ReactYield | ReactText | ReactFragment;
+import type { ReactPortal } from 'ReactPortal';
+
+export type ReactNode = ReactElement<any> | ReactCoroutine | ReactYield | ReactText | ReactFragment | ReactPortal;
```

Comment:
Can you place `ReactPortal` before `ReactText` and `ReactFragment`? It used to matter. I'm not sure it still does but there was something weird with `ReactEmpty` before. I think I had a similar comment in another PR.